### PR TITLE
[PPML][DOC] refine spark on k8s with EHSM docs

### PR DESCRIPTION
**Pull request by Uxito-Ada**
## Description

Fix wrong links, append missing steps, remove outdated parameters from PPML docs to make them more interpretable. 

### 1. Why the change?

Update PPML docs.

### 2. User API changes

None.

### 3. Summary of the change 

refine spark on k8s with EHSM docs

### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [ ] Application test
- [x] Document test
- [ ] ...

### CODE REVIEW

**Comment by Uxito-Ada** [comment]
Should be merged after #6360 .

File: ppml/README.md [hunk 1]
```diff
@@ -233,9 +231,69 @@ To build your own Big Data & AI applications, refer to [develop your own Big Dat
...
+       -v /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket \
+       -v $DATA_PATH:/ppml/trusted-big-data-ml/work/data \
+       -v $KEYS_PATH:/ppml/trusted-big-data-ml/work/keys \
+       -v $SECURE_PASSWORD_PATH:/ppml/trusted-big-data-ml/work/password \
+       -v $KUBECONFIG_PATH:/root/.kube/config \
+       -e RUNTIME_SPARK_MASTER=$K8S_MASTER \
+       -e RUNTIME_K8S_SPARK_IMAGE=$DOCKER_IMAGE \
+       -e LOCAL_IP=$LOCAL_IP \
+       $DOCKER_IMAGE bash
+    ```
```
Comment (liu-shaojun) [inline ppml/README.md:291]: It seems line234 - line291 could be put into prepare_environment.md as it has the same content.

Comment (Uxito-Ada) [reply ppml/README.md:291]: Maybe we should delete start-client-container section from prepare_environment.md. The prepare_environment is step 0 while the client container needs to be started after customer image building.

Comment (liu-shaojun) [reply ppml/README.md:291]: What about put start-client-container section in the end of **Step 1. Build your PPML image for production environment** (after create custom image) and update K8S RBAC section in prepare_environment.md?

Comment (Uxito-Ada) [reply ppml/README.md:291]: That's great.

**Review by liu-shaojun — APPROVED** [review]
LGTM

**Pull request merged**